src/control.cc

```diff
@@ -159,6 +160,13 @@ AbstractNode* ControlModule::getChild(const ValuePtr &value, const EvalContext*
 	return modulectx->getChild(n)->evaluate(modulectx);
 }
 
+static void applyChildren(const ModuleInstantiation *inst, AbstractNode *node, Context &c)
+{
+	inst->scope.apply(c);
+	std::vector<AbstractNode *> instantiatednodes = inst->instantiateChildren(&c);
+	node->children.insert(node->children.end(), instantiatednodes.begin(), instantiatednodes.end());
```

Comment:
since applyChildren() is always called on a fresh node, it's not necessary to copy, we can just do like the GroupNode: ```	node->children = inst->instantiateChildren(&c);```